### DISCUSSION TITLE
Help with correct settings for file comparison ignoring spacing.

### DISCUSSION THREAD

**Discussion by YanSkuratovich** [WinMerge/winmerge#2191] [Q&A]
So I need to compare two large sized files. 

example of 1st file:
abc123abc123abc123abc123abc123abc123abc123abc123abc123

example of 2nd file:
abc123
abc123
abc123
abc123
abc123
abc123
abc123
abc123
abc123

I am testing and one of the files has a text difference at the end of the file (on purpose). Playing with the settings it never shows me the difference in text and ignores it. 

![image](https://github.com/WinMerge/winmerge/assets/116044932/1a6732f7-f647-4ffa-b33d-d136ac383f1f)

In the image it should give me an error at ">>>" but it does not. at the bottom is says there is a difference found. If I press to show the difference it just marks everything red. 

These are my options:
![image](https://github.com/WinMerge/winmerge/assets/116044932/6cdbc2c3-309f-4de0-9287-dd5c7b3417d7)

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not do a good job of comparing files where such text is line-wrapped and non-wrapped.

However, by using the Replace plugin in the "Select Files or Folders" window as shown below, you can remove line breaks from the text and compare the text on both sides by breaking lines at specific character positions.

~~~
Replace -e "\r\n" "" | Replace -e "3" "3\r\n"
~~~
The above assumes that the new line is CRLF(\r\n), replaces the new line with an empty character, and inserts a new line after the character "3".


![image](https://github.com/WinMerge/winmerge/assets/98126/1e1fdabe-c3ba-4006-af6f-e08fc91d5ae2)
![image](https://github.com/WinMerge/winmerge/assets/98126/6b99b6a2-2f94-4ce3-9456-d8bc5aa41dda)

**Reply by YanSkuratovich** [reply]
Is there a different way to do it because in the actual file not all of the text is the same. so not every line ends in the same character.

**Reply by sdottaka** [reply]
Unfortunately, there is no other way than the above.
